Gemfile

```diff
@@ -143,7 +143,7 @@ group :test do
   gem "rspec-rails", "~> 2.99.0", :group => :development
   gem 'rspec-activemodel-mocks'
   gem 'rspec-example_disabler', git: "https://github.com/finnlabs/rspec-example_disabler.git"
-  gem 'capybara', '~> 2.3.0'
+  gem 'capybara'
```

Comment:
Because people tend to use "bundle update" instead of "bundle (install)", we tend to restrict the versions.